task07-consume-real-data/json-data.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary js with json related functions
+ */
+
+ function sortArray(array){
+    array.sort((a,b) => {
+        let a1 = a.name.toLowerCase(),
+            b1 = b.name.toLowerCase();
+    
+        if (a1 < b1) {
+            return -1;
+        }
+        if (a1 > b1) {
+            return 1;
+        }
+        return 0;
+    });
+}
+
+export async function fetchOnePieceData(){
+    const OnePieceUrl = 'https://static.akurey.com/trainings/OnePieceInformation.json';
+    
+    try {
+        let onePieceInfo = await fetch(OnePieceUrl)
+        .then( (response) => response.json())
+        .catch( (error) => console.log(error));
```

Comment:
Try to don't use console.log, in this case you can open a alert or something similar.